test/MusicStore.Test/AnnouncementComponentTest.cs

```diff
@@ -0,0 +1,107 @@
+﻿using System;
```

Comment:
MusicStore files don't have file headers, and I remember that you were fine with it. But, let me know if you want me to add file headers for all MusicStore files..